src/components/Table/TableRow.tsx

```diff
@@ -0,0 +1,91 @@
+import get = require('lodash/get');
+import isEqual = require('lodash/isEqual');
+import isFunction = require('lodash/isFunction');
+import map = require('lodash/map');
+import * as React from 'react';
+import { TableColumn } from 'rendition';
+
+/*
+ * Get the value specified by the `field` value
```

Comment:
Can you create a separate PR/Commit that extracts `TableRow` into its own file?

Reply:
I personally think that refactorings don't necessarily need a separate issue and PR as it is something we are doing continuously (or we should be), unless it is a large change that might affect functionality. I can do it of course, but even though it is a bit cleaner, I feel it is unnecessary additional work. Do you agree?

Reply:
@sradevski At the very least, separating the refactor into a different commit is good practice. A separate PR would make this much easier to review, right now it's very hard to see the difference between what was refactored and what was changed. Additionally, the refactor appears to be unrelated to the functionality change here.

Reply:
It is really just a copy-paste from the Table component into a new file, but I do agree it is a bit difficult to see when reviewing. I will separate it into its own commit.